packages/glimmer-runtime/lib/syntax/core.ts

```diff
@@ -241,16 +247,19 @@ export class Modifier implements StatementSyntax {
   }
 
   constructor(options) {
+    super();
     this.path = options.path;
-    this.params = options.params;
-    this.hash = options.hash;
+    this.args = options.args;
   }
 
-  evaluate(stack) {
-    return stack.createMorph(Modifier);
+  compile(compiler: CompileInto, env: Environment) {
+    compiler.append(new ModifierOpcode({ name: intern(this.path) }));
+  }
+
+  prettyPrint(): PrettyPrint {
+    return new PrettyPrint('modifier', this.path, [this.args.prettyPrint()]);
   }
```

Comment:
That's because we stopped needing to inspect the syntax objects. We just do the wire format and opcodes now.